### PR TITLE
Refactored HighlightCeption.php to remove dependency on javascript libraries

### PR DESCRIPTION
**Pull request by nshelms**
Took advantage of the native Facebook\Webdriver libraries to remove dependencies on jquery libraries. Doing this will make maintenance easier as libraries require updating to newer versions.

### CODE REVIEW

File: src/HighlightCeption.php [hunk 1]
```diff
@@ -27,10 +28,10 @@ class HighlightCeption extends Module
 
     /**
      * Css style for hightlight text or element
-     * 
+     *
      * @var array
```
Comment (arzzen) [inline src/HighlightCeption.php]: Please change  doctype to string

File: src/HighlightCeption.php [hunk 2]
```diff
@@ -63,8 +64,8 @@ public function _before(TestInterface $test)
 
         $this->webDriverModule = $this->getModule($this->config['module']);
         $this->webDriver = $this->webDriverModule->webDriver;
+        $this->cssStyle = "background-color: {$this->config['cssStyle']['background-color']}; color: {$this->config['cssStyle']['color']};";
```
Comment (arzzen) [inline src/HighlightCeption.php]: Why is locked css style only for "background-color" and text "color"?
It is not flexible. When I want to add more/other styles, I can not.

File: src/HighlightCeption.php [hunk 3]
```diff
@@ -85,102 +86,98 @@ public function _after(TestInterface $test)
...
-                    $this->webDriverModule->executeJs(sprintf('jQuery(document).xpath("%s").css(%s);', addslashes($locator), $this->cssStyle));
+                  $el = $this->webDriver->findElement(WebDriverBy::xpath($locator));
                 } else {
-                    $this->debug('[Highlight Selector] ' . Locator::humanReadableString($locator));
-                    $this->webDriverModule->executeJs(sprintf('jQuery("%s").css(%s);', addslashes($locator), $this->cssStyle));
-                }
+                  // assume css
+                  $el = $this->webDriver->findElement(WebDriverBy::cssSelector($locator));
+              }
+              $this->webDriver->executeScript("arguments[0].setAttribute('style', 'background-color: yellow; color: black;')", [$el]);
```
Comment (arzzen) [inline src/HighlightCeption.php]: Why the strictly defined style `'background-color: yellow; color: black;'`? - see [comment](https://github.com/hyperia-sk/highlight-ception/pull/3/files/31c38558235614a60363a99533d7b92b0efe1f73#diff-172821954fd472dc8ebb88458bacf276R67)

**Review by arzzen — CHANGES_REQUESTED** [review]

File: src/HighlightCeption.php (hunk 3 above)
Comment (nshelms) [reply src/HighlightCeption.php]: Forgot about this functionality. Added it back into the file. Thanks for pointing that out.

File: src/HighlightCeption.php (hunk 2 above)
Comment (nshelms) [reply src/HighlightCeption.php]: Forgot about this functionality. Added it back into the file. Thanks for pointing that out.

**Review by arzzen — APPROVED** [review]

**Comment by arzzen** [comment]
@nshelms Thanks for your first contribution to highlight-ception! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**